packages/core/src/loaders/imageLoader.ts

```diff
@@ -183,6 +196,126 @@ export function loadAndCacheImages(
   return allPromises;
 }
 
+/**
+ * Loads an image given an imageId and optional priority and returns a promise
+ * which will resolve to the loaded image object or fail if an error occurred.
+ * The image is stored in the cache.
+ *
+ * @param imageId -  A Cornerstone Image Object's imageId
+ * @param options - Options to be passed to the Image Loader
+ *
+ * @returns Image Loader Object
+ */
+export function createAndCacheDerivedImage(
+  imageId: string,
```

Comment:
For consistency with the other calls, this should be the destination imageId, and the options should contain the referencedImageId - otherwise it is really easy to confuse things.

Reply:
the options is only for name, I changed the first argument to referencedImageId